### PR TITLE
docs: Update READMEs

### PR DESCRIPTION
**Pull request by arbrandes**
Update both the repository's README.rst and the template to reflect the current requirements as per [OEP-55](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055/decisions/0003-readme-specification.html).

### CODE REVIEW

**Review by brian-smith-tcril — COMMENTED** [review]
Overall this looks great! Thank you for putting this together!

I left a few comments with thoughts/questions on here, I'd love to hear what others think!

File: README-template-frontend-app.rst [hunk 1]
```diff
@@ -1,75 +1,256 @@
-|Build Status| |Codecov| |npm_version| |npm_downloads| |license| |semantic-release| ## As many of these as you'd like to include, or others if you wish!
+frontend-app-[PLACEHOLDER]
+#############################
 
-frontend-app-[YOUR NAME HERE]
-==============================
+.. note::
```
[the remark this answers is not in the document]
Comment (arbrandes) [reply README-template-frontend-app.rst:4]: Good point.  I'll try to find an alternative that Github _does_ support.

File: README-template-frontend-app.rst [hunk 2]
```diff
@@ -1,75 +1,256 @@
...
-------------
+   What is this MFE?  Add a 2-3 sentence description of what it is and what it
+   does.
+
+This is the Awesome MFE.  It was built to provide an unmatched learning
+experience, with improved tools for both randomized goodness and the ability to
+directly reference amaze-blocks in existing courses. This experience is powered
+by the new Fantastico storage engine.
+
+Getting Started
```
[the remark this answers is not in the document]
Comment (arbrandes) [reply README-template-frontend-app.rst:28]: Agreed.  I'll break those out!

File: README-template-frontend-app.rst [hunk 3]
```diff
@@ -1,75 +1,256 @@
...
+
+.. _Backstage: https://open-edx-backstage.herokuapp.com/catalog/default/component/frontend-app-[PLACEHOLDER]
 
-* A bulleted list of roadmap items, ideally with JIRA ticket numbers attached (but could just be
-  a high-level overview of upcoming features/epics)
+Reporting Security Issues
+*************************
 
-* OR - a link to a publically available wiki/document
+Please do not report security issues in public, and email security@axim.org instead.
```
Comment (nedbat) [inline README-template-frontend-app.rst:241]:
```suggestion
Please do not report security issues in public. Email security@axim.org instead.
```

File: README.rst [hunk 4]
```diff
@@ -1,76 +1,199 @@
...
+The assigned maintainers for this component and other project details may be
+found in `Backstage`_. Backstage pulls this data from the ``catalog-info.yaml``
+file in this repo.
+
+.. _Backstage: https://open-edx-backstage.herokuapp.com/catalog/default/component/frontend-template-application
+
+Reporting Security Issues
+*************************
+
+Please do not report security issues in public, and email security@axim.org instead.
```
Comment (nedbat) [inline README.rst:185]:
```suggestion
Please do not report security issues in public. Email security@axim.org instead.
```

File: README-template-frontend-app.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (arbrandes) [reply README-template-frontend-app.rst:4]: Apparently there is no equivalent functionality that Github supports.  I removed it from this repo's actual README, but left the ones in the template in.  After all, those notes are intended to be removed by the developer anyway, and will only be read outside Github.

File: README-template-frontend-app.rst [hunk 5]
```diff
@@ -1,75 +1,256 @@
...
+   .. code-block::
+
+      DEFAULT_SERVICES ?= \
+      lms
+
+#. Start the devstack with:
+
+   .. code-block::
+
+      cd devstack
```
[the remark this answers is not in the document]
Comment (arbrandes) [reply README-template-frontend-app.rst:83]: I tried it, but apparently it doesn't work well nested under a bullet point.  (The block doesn't respect formatting.)  So I guess I'll have to leave the explicit `code-block`s in.

**Comment by arbrandes** [comment]
Addressed comments!  If nobody has any further comments or objections by Monday, I'll merge this then.  Thanks!

**Pull request merged**